wave-fe/src/app/round.service.ts

```diff
@@ -16,29 +18,64 @@ export class RoundService {
   private privateAddr = `roundsPrivate/${this.roundId}`;
   private roundRef = firebase.database().ref(this.roundAddr);
   private roundSubject = new ReplaySubject<any>(1);
-  private newRoundData: any = null;
   private score: any = null;
+  private newRoundData: {
```

Comment:
Added for documentation purposes, Its easier to understand what the initial state is.